tests/homework4/test_task2.py

```diff
@@ -0,0 +1,27 @@
+import os
+import urllib.request
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from homework4.task2 import count_dots_on_i
+
+
+@patch("urllib.request.urlopen")
+def test_count_dots_on_i(mock_request):
+    with open(
+        os.path.dirname(os.path.abspath(__file__)) + "/wiki.html", "r"
+    ) as input_file:
+        html = input_file.read().encode()
+    cm = MagicMock()
```

Comment:
It's better to move the creation of MagicMock object into a fixture, together with file reading above